unified-codes/libs/ui/src/lib/components/inputs/molecules/ToggleButton.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+
+import Button, { ButtonProps } from '../atoms/Button';
+import AddIcon from '../../icons/atoms/AddIcon';
+import CheckCircleIcon from '../../icons/atoms/CheckCircleIcon';
+
+const styles = {
+  toggleButtonPrimary: {
+    backgroundColor: '#2B83A1',
+  },
+  toggleButtonSecondary: {
+    backgroundColor: '#5CCDF4',
+  },
+};
```

Comment:
I wondered about shifting the colours to the theme. 
Turns out that it is a bit of work and requires updating this file, the `muiTheme` and the `ToggleButtonStories.tsx` as well. 🙄 

The suggestion didn't show the diff very well - I could commit changes to show what I mean instead, if you like. Or we just ignore, which is fine. What I've done is nice and it means that the colours can be reused elsewhere. Aside from that.. no big benefit.

Reply:
Ah, I'd be interested to see! Sounds useful

Reply:
965ac822c with the addition of the `ThemeProvider` in the story only required because the theme isn't provided automatically in the storybook context.

In the web, that just works ( it is passed down in `index.tsx` )

Reply:
Nice!